r/tests/testthat/test-compressed.R

```diff
@@ -18,6 +18,8 @@
 context("arrow::io::Compressed.*Stream")
 
 test_that("can write Buffer to CompressedOutputStream and read back in CompressedInputStream", {
+  skip("Unsupported")
```

Comment:
Is this universally unsupported ? It probably has worked at some point ? 
Can you expand on what's happening ?

Reply:
The compression library was not linking in Windows... we could also consider throwing a "not supported" error message in the actual R file when run from Windows, or document this limitation properly.

Reply:
I guess that can be a feature flag. We can skip altogether for a first release I guess.

Reply:
Scoped the test to Windows, throwing an error for Windows and added details on the the function docs explaining this is not supported in Windows.